peerconnection_go_test.go

```diff
@@ -43,6 +44,107 @@ func (api *API) newPair(cfg Configuration) (pcOffer *PeerConnection, pcAnswer *P
 	return pca, pcb, nil
 }
 
+const liteOffer = `v=0
```

Comment:
I would cut this down to the bare minimum. I think the follow should be enough to confirm behavior.

```
v=0
o=- 1613490992396387 1 IN IP4 123.45.67.89
s=VideoRoom 5538594542389977
t=0 0
a=group:BUNDLE
a=msid-semantic: WMS janus
a=ice-lite
```

Reply:
had to add a few extra lines, I used minimal offer from another test file and trimmed it down.